### ISSUE TITLE
"Image is missing alt text uses" <a> as example

### ISSUE THREAD

**Issue by snorrekim** [Khan/tota11y#3]
It should use ```<img src="..." alt="A description of the image"></img>``` as example.
![image](https://cloud.githubusercontent.com/assets/5329330/8059494/8e672cc8-0ec0-11e5-9810-b3db421f2a5d.png)

**Comment by jdan** [comment]
Yep this has been fixed :)

https://github.com/Khan/tota11y/commit/558e7b8c95786e7ffb2975379b93d7ad3fab53e9

Updating on the project page shortly.
